cips/cip-x.md

```diff
@@ -50,7 +50,8 @@ func estimateTail(head, tail Header, blockTime, pruningWindow time.Duration) (he
 |---------------------|----------------|-----------------------------------------------|---------------------------|
 | HeaderPruningWindow | 30 days        | Defines the length of time to retain headers  | No                        |
 
-By default, HeaderPruningWindow equals to SamplingWindow from [CIP-004](./cip-004.md), but its configurable per LN.
+By default, HeaderPruningWindow equals to SamplingWindow from [CIP-004](./cip-004.md). It can be modified per LN, but
+it must be more or equal to the SamplingWindow.
```

Comment:
Do you want to clarify why we don't want to allow LN to have pruning window < sampling window?

Reply:
> As there is currently way to sync history beyond HPW, the HPW less than SPW is gonna fail sampling. SPW is currently 30 days.

From notion. Doesn't hurt to add this here

Reply:
[ceed598](https://github.com/celestiaorg/CIPs/pull/279/commits/ceed5986e1e2cf9bedad74dda89455957bead4ed)